### PR TITLE
Add option to remove slash to `normalize_path_middleware`

### PR DESCRIPTION
**Pull request by kryptn**
## What do these changes do?

This change adds new functionality to the `normalize_path_middleware`. I wanted the option to remove trailing slashes and it made sense to just add the functionality to the existing middleware.

## Are there changes in behavior for the user?

No breaking changes to existing usage. The `normalize_path_middleware` factory only accepts keyword arguments, and because there's now an assertion that only one of `append_slash` and `remove_slash` can be enabled this change shouldn't be breaking.

## Related issue number

None that I could find.

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

**Comment by kryptn** [comment]
Thanks! Can do, but I'm not sure which version to add in there. should it be the expected release or the current version i made the change to?

**Comment by kryptn** [comment]
Went ahead and assumed the next minor version and updated.

Thanks again!

File: docs/web_reference.rst [hunk 1]
```diff
@@ -2762,27 +2762,41 @@ Normalize path middleware
...
 .. function:: normalize_path_middleware(*, \
-                                        append_slash=True, merge_slashes=True)
+                                        append_slash=True, \
+                                        remove_slash=False, \
+                                        merge_slashes=True, \
+                                        redirect_class=HTTPMovedPermanently)
 
-  Middleware factory which produces a middleware that normalizes
-  the path of a request. By normalizing it means:
+   Middleware factory which produces a middleware that normalizes
```
Comment (asvetlov) [inline docs/web_reference.rst:2770]: In sphinx the normal indentation is 3 spaces.
We use this default everywhere in aiohttp docs.
Why did you change it?

Comment (kryptn) [reply docs/web_reference.rst:2770]: The indentation in that section was two, it's now three. I noticed the difference when adding the `.. versionadded::` tag.

Comment (asvetlov) [reply docs/web_reference.rst:2770]: Got it. Sorry, spaces are hard to count in github review tool

Comment (kryptn) [reply docs/web_reference.rst:2770]: No problem, I even had to open it back up in my editor to verify because it did look like four in github. 

Thanks!
